### ISSUE TITLE
Batch Transfer and Tracking of manga

### ISSUE THREAD

**Issue by Doomsdayrs** [inorichi/tachiyomi#1729]
**App version:**
0.8
**Request:**
Ability to batch move mangas from one source to another. This can also be applied to batch tracking via trackers. 

**Other details:**
The idea would be the app would automatically (transfer/ track) mangas with exact naming. And anything that doesn't have an exact match would be cached up to be sorted through immediately after the automatic process. 

The manual process would work like the pop up window of the tracker. Where it lists the mangas that are possible matches. But instead of a title on top, it would get the data of the manga you currently have in a card to the left. Then the user would select the matching manga, confirm it, and then be presented with the next one needing manual entry (or it ends and confirms that the process is over). The user could also choose to skip over the manga if they wished.

The above could be done in idea with a simple loop and if statement that attempts to get a direct match, if there isn't it put the specific index of the manga (lets say manga 0) into another array to be sorted out by the manual process.

**Comment by metroid112** [comment]
Have you tried Source Migration?, go to your Library -> 3 dots menu at top right -> Source Migration

**Comment by Samk19** [comment]
Need batch transfer of manga from one source to another....

**Comment by Doomsdayrs** [comment]
I changed my idea to be more specific on how this could be accomplished, i don't know Kotlin so i can't figure how to do it in tachiyomis environment

**Comment by hXtreme** [comment]
For migration, it seems to be a problem with how tachiyomi has implemented migration system. The migration logic and the UI seem to be strongly coupled, with the logic part calling functions (migrate/migrateManga) from UI part and vice versa(SearchController or something of the sort).

That's kind of what is giving me a bit of an issue with solving #1903

**Comment by kaononymous** [comment]
Yes! Have always wanted this feature 😊 Seems difficult but will be eternally grateful to see this. Well explained @Doomsdayrs

**Comment by kondanta** [comment]
Mind if I ask the current status of this issue?

**Comment by hXtreme** [comment]
> Mind if I ask the current status of this issue?

Right now no one seems to be working on this and you should feel free to try to resolve this.

**Comment by ImMapple** [comment]
tachiyomij2k has this feature and i wonder when tachiyomi will have it

**Comment by jobobby04** [comment]
Never at this rate, the code is spaghetti, so it won't be added to main Tachi.

**Comment by Mtkmanjaro** [comment]
.

**Comment by maxdd** [comment]
Up, please add the feature especially with the recent errors everybody are receiving

**Comment by kingtie** [comment]
Hey, so, this feature is becoming more and more needed as time goes on. I've had to migrate Asura Scans, Reaper scans and others one at a time back to the same exact source multiple times now. I don't know if it's because they're changing the addresses to their website or if the extensions just need updates.
